### DISCUSSION TITLE
Host User creation.

### DISCUSSION THREAD

**Discussion by srelf-eschercloud** [gravitational/teleport#18157] [Q&A] [answered]
Following this guide, is it possible to set the shell that should be used by the user

https://goteleport.com/docs/server-access/guides/host-user-creation/

As currently, it drops me into a /bin/sh.

Cheers
Steve.

**Comment by webvictim** [comment, marked as answer]
It isn't currently possible to control this directly via Teleport's configuration.

Teleport just runs `useradd` under the hood to actually add the user, which generally adheres to whatever is set in `/etc/default/useradd` (although it can be a little distro specific). At any rate, adding `SHELL=/bin/bash` to `/etc/default/useradd` should change the default shell.

**Reply by srelf-eschercloud** [reply]
makes sense, thanks
